mesonbuild/compilers/compilers.py

```diff
@@ -388,6 +388,16 @@ def get_base_link_args(options: 'KeyedOptionDictType', linker: 'Compiler',
 class CrossNoRunException(MesonException):
     pass
 
+
+class FindLibraryResult(HoldableObject):
+    def __init__(self, linkargs: T.Optional[T.List[str]], cached: bool = False):
+        if linkargs is not None:
+            self.linkargs = linkargs.copy()
+        else:
+            self.linkargs = None
```

Comment:
```suggestion
        self.linkargs = linkargs.copy() if linkargs else None
```

Reply:
This does not fully have the same semantics as the original code, but I agree it's better.